smac/optimizer/local_bo/rh2epm_boing.py

```diff
@@ -0,0 +1,92 @@
+import typing
+
+import copy
+
+import numpy as np
+
+from smac.runhistory.runhistory import RunHistory
+from smac.runhistory.runhistory2epm import (
+    RunHistory2EPM4Cost,
+    RunHistory2EPM4LogScaledCost,
+)
+
+
+class RunHistory2EPM4CostWithRaw(RunHistory2EPM4Cost):
```

Comment:
can you move that to the runhistory directory?

Reply:
This class return 3 items instead of 2 in `RunHistory2EPM4Cost` and thus can only be used by BOinG, I would rather keep it here to avoid confusion.